plugins/modules/info.py

```diff
@@ -3110,6 +3492,16 @@ def perform_module_operation(self):
         if 'support_assist_settings' in str(subset):
             support_assist_settings = SupportAssist(
                 self.support_assist_api, self.module).get_support_assist_settings()
+        if 'alert_settings' in str(subset):
```

Comment:
There are so many if statements and no sonar issues on complexity, Can we please check this.
@meenakshidembi691  @trisha-dell

Reply:
@sachin-apa 
Addressed and fixed the CC issues for info module.